### ISSUE TITLE
State not saved to database

### ISSUE THREAD

**Issue by umutnacak** [darian311/VillageMakeover#21]
Sometimes I don't see my last placed object after refreshing web page.
Game posts correct data to /api save_state and gets 200 OK.
Google app engine console logs /api save_state entry.
But the correct state is not in database.

Did anybody see something like this? I'm testing on umutvillage and localhost. I hope this is because of those environments. Otherwise there is something wrong on server side.

**Comment by darianhickman** [comment]
No that's the first time hearing of it. Is this reproducable pretty easily?

On Wed, Mar 25, 2015 at 4:16 PM, umutnacak notifications@github.com wrote:

> Sometimes I don't see my last placed object after refreshing web page.
> Game posts correct data to /api save_state and gets 200 OK.
> Google app engine console logs /api save_state entry.
> But the correct state is not in database.
> 
> Did anybody see something like this? I'm testing on umutvillage and
> localhost. I hope this is because of those environments. Otherwise there is
> something wrong on server side.
> 
> —
> Reply to this email directly or view it on GitHub
> https://github.com/darian311/VillageMakeover/issues/21.

## 

Darian Hickman |  (310) 439-8573  | http://www.linkedin.com/in/darianhickman

**Comment by umutnacak** [comment]
Yes just place an object and refresh the page. But it happens one time out
of 10 or maybe less(less means 1 time out of 15).
